epochStart/bootstrap/simpleShardHeaderInterceptor.go

```diff
@@ -0,0 +1,37 @@
+package bootstrap
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type simpleShardHeaderInterceptor struct {
```

Comment:
we might not require this interceptor as it is of interest only the metachain

Reply:
we can delete it if we don't need it